baseline-refaster-javac-plugin/baseline-class-uniqueness.lock

```diff
@@ -2,17 +2,17 @@
 # Run ./gradlew checkClassUniqueness --write-locks to update this file
 
 ## runtimeClasspath
-[com.google.auto.service:auto-service-annotations, com.google.errorprone:error_prone_core, com.google.errorprone:error_prone_refaster]
```

Comment:
Seems to be a behavior change in https://github.com/palantir/gradle-baseline/pull/1837 and now `[com.google.auto.service:auto-service-annotations` is no longer getting reported.

Reply:
Didn't see any other complaints. The class diff might be due to other update PRs that are not part of the changelog.